samcli/hook_packages/terraform/hooks/prepare/hook.py

```diff
@@ -1434,7 +1438,7 @@ def _get_makefile_build_target(logical_id: str) -> str:
     str
         The formatted Makefile rule build target
     """
-    return f"build-{logical_id}:{os.linesep}"
+    return f"build-{logical_id}:\n"
```

Comment:
make our generated make file in a unix format, so it can be executed in all Oss and also in container from a windows machine